payment_payfip/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,3 @@
+This module adds the payment acquirer PayFIP.
+
+* Official site: https://www.collectivites-locales.gouv.fr/payfip
```

Comment:
FYI, that url is not valid. 404.

Reply:
Indeed, they renewed their website and now there's a dedicated domain for payfip.
I updated the readme

Reply:
Thanks